addons/web/static/tests/legacy/fields/relational_fields_tests.js

```diff
@@ -43,7 +49,9 @@ QUnit.module('relational_fields', {
                     user_id: {string: "User", type: 'many2one', relation: 'user'},
                     reference: {string: "Reference Field", type: 'reference', selection: [
                         ["product", "Product"], ["partner_type", "Partner Type"], ["partner", "Partner"]]},
-                    model_id: {string: "Model", type:'many2one', relation:'ir.model'}
+                    model_id: {string: "Model", type:'many2one', relation:'ir.model'},
+                    stage: {string: "Stage", type: "selection", searchable:true,
```

Comment:
there's already a selection field, no need to add a new one (for everyone on top of that)

Reply:
This is a clear indication to me that you did what you did for a restrictive scope.